### ISSUE TITLE
Image and text in the intro sections of the parts get superimposed in certain conditions

### ISSUE THREAD

**Issue by VPuosk** [fullstack-hy2020/fullstack-hy2020.github.io#1558]
With (tested using Edge & Chrome) screen widths between 640 and 991 some of the graphical elements on the pages - especially at the beginning of pages have layout issues. As in being rendered on top of each other. This seems to be just a visual issue. The part of the page referred to seems to work fine at screen widths <640 and >991

![image](https://user-images.githubusercontent.com/57898066/139185286-53e08f4a-ff4a-44e0-9f6c-4766d0ec9fb9.png)

**Comment by KartikAroraOfficial** [comment]
Hey, there doesn't seem to be any such issue. I checked it with Both chrome and Edge. it seems to be working fine.

**Comment by KartikAroraOfficial** [comment]
You're correct, this needs to be looked into, But I don't think it's much of an issue since most users here are on laptop and rarely anyone would be using their iPad Mini for this, but I'll keep this in mind.

**Comment by mluukkai** [comment]
Material will be migrated elsewhere, so this will no longer be relevant
